### PR TITLE
chore: Adds reset and build scripts without running tests

### PR DESCRIPTION
**Pull request by jorbuedo**
## Description / Change(s) / Related issue(s)

`yarn reset:dev` to clean install everything without running tests.

### CODE REVIEW

File: packages/api/package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
   ],
   "scripts": {
     "build": "yarn tsc && yarn lint && yarn test --ci --silent && yarn clean && bob build",
+    "build:dev": "yarn clean && bob build",
```
Comment (michalszorad) [inline packages/api/package.json]: `yarn tsc` as well?

Comment (michalszorad) [reply packages/api/package.json]: Same applies below in other packages

Comment (jorbuedo) [reply packages/api/package.json]: I don't know, it's adding more time. Depends if we care about tsc when running this.

Comment (michalszorad) [reply packages/api/package.json]: I'd say to keep tsc as well, so it's easier to spot TS bugs locally

**Review by michalszorad — APPROVED** [review]

**Comment by stackchain** [comment]
@jorbuedo bringing in after rebasing.

**Pull request merged**